pinot-common/src/main/java/org/apache/pinot/common/function/FunctionRegistry.java

```diff
@@ -216,5 +216,10 @@ public static Object clpDecode(String logtypeFieldName, String dictVarsFieldName
         String defaultValue) {
       throw new UnsupportedOperationException("Placeholder scalar function, should not reach here");
     }
+
+    @ScalarFunction(names = {"arrayToMultiValue", "array_to_multi_value"}, isPlaceholder = true)
+    public static String arrayToMultiValue(Object multiValue) {
```

Comment:
(minor) Personally prefer `arrayToMV` to be consistent with other MV functions such as `sumMV`

Reply:
done